### ISSUE TITLE
Add Vaadin Elements to Demo Kitchen Sink

### ISSUE THREAD

**Issue by hotforfeature** [hotforfeature/origami#18]
[Vaadin Elements](https://vaadin.com/elements) are easily some of the most used Polymer-built elements outside of Google's own collections. While Origami explicitly supports Google's collections, it'd be good to showcase other elements developers may want to use that Origami doesn't have a collections library for.

**Comment by hotforfeature** [comment]
Pushed an update to the vaadin branch that has `<vaadin-grid>` working. Only thing that's wonky is two-way databinding for the select all checkbox, and that's part of a bigger issue with Angular properties getting bound to Polymer templates that I created #36 for.

If you're watching this issue for vaadin support, most elements that I've tried work naturally. The grid will depend on vaadin/vaadin-grid#890 getting merged, so subscribe to that if you're looking for grid support.

This issue will remain open for a little while since I'd like to clean up the demo project and organize it better.
